src/chrome/content/rules/Aeriagames.xml

```diff
@@ -1,37 +1,16 @@
 <!--
-	CDN buckets:
-
-		- wildcard.aeriastatic.com.edgekey.net
-
-			- c.aeriastatic.com
-
-		- static.aeriagames.com.edgesuite.net
-
-			- s.aeriastatic.com
-
-
-	Problematic domains:
-
-		- s.aeriastatic.com	(works, akamai)
+	CDN buckets
 
+		Updated CDN
+			- cms-content.s.aeriastatic.co
 -->
 <ruleset name="Aeriagames">
-<target host="www.aeriagames.com" />
-<target host="aeriagames.com" />
+	<target host="www.aeriagames.com" />
+	<target host="aeriagames.com" />
 	<target host="*.aeriastatic.com" />
-
-<rule from="^http://(?:www\.)?aeriagames\.com/" to="https://www.aeriagames.com/"/>
-
-	<rule from="^http://c\.aeriastatic\.com/"
-		to="https://c.aeriastatic.com/" />
-
-	<!--	We must rewrite from https://s to c due to protocol-relative
-		links on www.aeriagames.com:
-
-			- https://mail1.eff.org/pipermail/https-everywhere-rules/2013-July/001650.html
-			- https://bugzilla.mozilla.org/show_bug.cgi?id=879072#c1
-							-->
-	<rule from="^https?://s\.aeriastatic\.com/"
-		to="https://c.aeriastatic.com/" />
-
+		<test url="http://cms-content.s.aeriastatic.com/82de179009b2505fcc419f60f4d98ab7/files/aurakingdom/image/F/Featured_Shinobi.jpg" />
+		<test url="http://c.aeriastatic.com/modules/js/AG/version1/assets/signupVerifier.css" />
+		<test url="http://cms-content.s.aeriastatic.com/cb69c2e1a7010ab58379cbef75909305/files/eos/image/e/episode_2.5_EOS_featured_2.jpg" />
+	<rule from="^http:"
+		to="https:" />
```

Comment:
The generic redirect is a bit dangerous in combination with the wildcard target above.